popcoder/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.1.1'
```

Comment:
I recommend that you don't edit the version as part of a PR. 
Imagine someone else (e.g. you) is working on another PR that takes time to merge. You'd get a horribly conflict on the version.